src/askui/models/openrouter/handler.py

```diff
@@ -0,0 +1,77 @@
+import os
```

Comment:
I think we should name the file `model.py` or `get_model.py`. This `handler.py` from `anthropic` is a bit misleading and inconsistent with the rest of the naming. WDYT?